.travis.yml

```diff
@@ -18,6 +18,7 @@ matrix:
     - os: osx
       osx_image: xcode9.3
       env: PYTHON_VERSION=python3 ONNX_ML=0
+      python: "3.6"
```

Comment:
this works for Linux but is ignored on macOS or Windows
Check this: https://docs.travis-ci.com/user/languages/python/#running-python-tests-on-multiple-operating-systems

Reply:
I think this is where you need to make the change :
https://github.com/onnx/onnx/blob/ba76e4508eecf08797c2c5b7b9ca196fabbeab0f/.travis/before_install.sh#L31